scss/_progress.scss

```diff
@@ -1,32 +1,37 @@
+// Progress animations
 @keyframes progress-bar-stripes {
   from { background-position: $progress-height 0; }
   to { background-position: 0 0; }
 }
 
+// Basic progress bar
 .progress {
   display: flex;
+  align-items: center;
+  height: $progress-height;
   overflow: hidden; // force rounded corners by cropping it
   font-size: $progress-font-size;
   line-height: $progress-height;
-  text-align: center;
   background-color: $progress-bg;
   @include border-radius($progress-border-radius);
-  @include box-shadow($progress-box-shadow);
 }
 
 .progress-bar {
-  height: $progress-height;
-  line-height: $progress-height;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
```

Comment:
This isn't necessary as far as I can tell. See https://jsbin.com/wahusavehu/edit?html,css,output.